docker-compose-services/drupalci-chromedriver/docker-compose.chromedriver.yaml

```diff
@@ -12,9 +16,13 @@ services:
     links:
       - chromedriver:$DDEV_HOSTNAME
     environment:
-      # Pick if you want to run tests as sqlite or mysql and uncomment.
+      # *** One of these must be uncommented ***
+      # In order for the system to work, one of these must be uncommented so
+      # that the test system can connect to the database.
       # - SIMPLETEST_DB=sqlite://tmp/test.sqlite
       # - SIMPLETEST_DB=mysql://db:db@db:3306/db
+      #
+      # Note: Do not modify the base URL value.
       - SIMPLETEST_BASE_URL=http://web
```

Comment:
The suggestion above makes "web" be unambiguous, so this recipe can work with other projects running.